src/contracts/pods/EigenPod.sol

```diff
@@ -437,20 +437,24 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
         IEigenPod.VerifiedPartialWithdrawal calldata verifiedPartialWithdrawal,
```

Comment:
would name this struct something different... like VerifiedPartialWithdrawalBatch?